packages/dds/tree/src/test/feature-libraries/sequence-field/rebase.spec.ts

```diff
@@ -334,4 +364,41 @@ describe("SequenceField - Rebase", () => {
         const expected = Change.insert(1, 1);
         checkDeltaEquality(insertE2.change, expected);
     });
+
+    it("concurrent insert and move ↷ delete", () => {
+        const delA = tagChange(Change.delete(0, 1), brand(1));
+        const insertB = tagChange(Change.insert(0, 1), brand(2));
+        const moveC = tagChange(Change.move(2, 1, 1), brand(3));
```

Comment:
If all three operations are concurrent, wouldn't C be at index 1?

Reply:
I don't understand the question, but C is moving a node to index 1.

Reply:
I was assuming a starting state [ACD]. I'm guessing it's [ADC] instead.
It seems like the end result is [BCD], shouldn't it be [CBD] instead since the move-in is sequenced after the insert and everything has merge-left policy?

Reply:
I don't follow your notation, but the intention for this test was to check that we are able to retain the knowledge that the move-in goes to the right of the insert (since they are in the same position after rebasing over the delete).

Reply:
Oh I get it. For what it's worth, I've been using "A", "B", "C", etc. to refer to nodes in the document, where the nodes are expected to end up in alphabetical order. I got confused about the test because I trusted that convention when I shouldn't have. It seems like you're just using the letters to refer to the changes instead.
Anyway, mystery solved. :)